### PR TITLE
test inc update - added errata applicability

### PR DESCRIPTION
**Pull request by pondrejk**
Aims to prevent the corner case from #6012

### CODE REVIEW

File: tests/foreman/longrun/test_inc_updates.py [hunk 1]
```diff
@@ -201,6 +202,8 @@ def setUp(self):
                          ' and resource_id = {}'
                          ' and started_at >= {}'.format(host[0].id, timestamp)
         )
+        # Force host to generate or refresh errata applicability
+        call_entity_method_with_timeout(host.errata_applicability)
```
Comment (ldjebran) [inline tests/foreman/longrun/test_inc_updates.py]: perhaps add more timeout value as the default one is 300 sec 
for example 
```
call_entity_method_with_timeout(host.errata_applicability, timeout=600)
```

**Comment by pondrejk** [comment]
@ldjebran test result:
```
tests/foreman/longrun/test_inc_updates.py -k test_positive_noapply_api

============================= test session starts ==============================
platform linux -- Python 3.6.5, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /home/jenkins/shiningpanda/jobs/ad0ef6b5/virtualenvs/d41d8cd9/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jenkins/workspace/satellite6-standalone-automation@2, inifile:
plugins: wait-for-1.0.9, xdist-1.22.2, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collecting ... collected 2 items
2018-06-05 10:25:01 - conftest - DEBUG - Collected 2 test cases


tests/foreman/longrun/test_inc_updates.py::IncrementalUpdateTestCase::test_positive_noapply_api PASSED [100%]

 generated xml file: /home/jenkins/workspace/satellite6-standalone-automation@2/foreman-results.xml 
============================== 1 tests deselected ==============================
=================== 1 passed, 1 deselected in 271.79 seconds ===================
```
...this is from standalone automation job 1227.
Please also consider the updated cherrypick
